### PR TITLE
Gh-264: Fix links in HTML to point to the new docs site

### PR DESCRIPTION
**Pull request by GCHQDeveloper314**
This is a hotfix, the source for the pages will be correct in the next Gaffer release.

### CODE REVIEW

File: latest/swagger-doc/v2/properties.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "gaffer.properties.app.title": "Gaffer REST API",
-  "gaffer.properties.app.doc.url": "https://gchq.github.io/gaffer-doc/"
-}
\ No newline at end of file
+  "gaffer.properties.app.doc.url": "https://gchq.github.io/gaffer-doc/v1docs/"
```
Comment (t92549) [inline latest/swagger-doc/v2/properties.json:3]: What does this do?

File: v1docs/swagger-doc/v2/properties.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
   "gaffer.properties.app.title": "Gaffer REST API",
-  "gaffer.properties.app.doc.url": "https://gchq.github.io/gaffer-doc/"
-}
\ No newline at end of file
+  "gaffer.properties.app.doc.url": "https://gchq.github.io/gaffer-doc/v1docs/"
```
Comment (t92549) [inline v1docs/swagger-doc/v2/properties.json:3]: What does this do?

Comment (GCHQDeveloper314) [reply v1docs/swagger-doc/v2/properties.json:3]: It's used by part of Swagger Doc. The property is [documented here in the Javadoc](https://gchq.github.io/gaffer-doc/latest/javadoc/gaffer/constant-values.html) as `APP_DOCUMENTATION_URL`. It seems this should be set to the URL which is used to host the page. Although I'm not convinced it really makes any difference as the Swagger Doc is working fine with it set to the wrong value.

File: latest/swagger-doc/v2/properties.json (hunk 1 above)
Comment (GCHQDeveloper314) [reply latest/swagger-doc/v2/properties.json:3]: See other comment. The `latest` and `v1docs` folders are identical. The latest folder is what is served by default, once v2 docs are deployed, only `v1docs` will contain the old docs.

**Pull request merged**